config/secrets.yml.example

```diff
@@ -19,6 +19,10 @@ development:
       secret: <%= ENV["OPENSTAX_ACCOUNTS_SECRET"] %>
       url: <%= ENV["OPENSTAX_ACCOUNTS_URL"] || 'http://localhost:2999' %>
       stub: <%= ENV["OPENSTAX_ACCOUNTS_STUB"] %>
+  exceptions:
```

Comment:
make all three sections look like what I did in tutor-server and exchange (verbatim)